### ISSUE TITLE
count/amount PATO_0000070

### ISSUE THREAD

**Issue by wdahdul** [pato-ontology/pato#65]
PATO_0000070 (formerly 'count') was recently renamed 'amount'. The new label 'amount' makes our existing annotations awkward sounding (for example, 'dorsal fin ray', amount, 6-10)

I think we can distinguish between count (a number of things that are relatively easy to count, e.g., fin ray count) vs. amount (a number that is difficult to count in discrete units, e.g., amount of blood). Can we create the sibling terms 'count' and 'amount'?

**Comment by cmungall** [comment]
This seems to be getting into units.

Unfortunately count wouldn't work for some of the existing uses of the class.

**Comment by austinmeier** [comment]
We (Planteome group) would like to use 'amount' for things not in reference to the whole organism, as stated in the current 'amount' definition:  

"The number of entities of this type that are part of the whole organism."

would suggest changing the definition to:

"The number of entities of this type that are part of another entity." or something similar.  

We are often quantifying the amount of entities/leaf, or entities/inflorescence et cetera.

**Comment by cmungall** [comment]
How about 'quantity'

**Comment by cooperl09** [comment]
I agree with Wasila's comment above that the label 'amount' here is confusing to most everyone, and Austin's comment about the definition.  
Amount is generally used with mass nouns, not count nouns.


If we are going to use this to designate the number of an entity, it would be better to call it "count" or 'number_of' or quantity (as Chris suggests above). 
Also, it is not clear why it is in the qualitative branch- this adds to the confusion. Propose it should be moved to be is_a physical object property

Propose revised name and definition: 
number_of (PATO:0000070): The count (or quantity) of the entities of this type. 

Also, this makes sense in terms of the reciprocal quality 'has_number_of'

@cmungall @gkoutos
